src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -225,9 +225,14 @@ private void startPostmaster() throws IOException
         final ProcessBuilder builder = new ProcessBuilder(args);
 
         builder.redirectErrorStream(true);
-        builder.redirectError(outputRedirector);
+        builder.redirectError(errorRedirector);
         builder.redirectOutput(outputRedirector);
-        postmaster = builder.start();
+        final Process postmaster = builder.start();
+
+        if (outputRedirector.type() == ProcessBuilder.Redirect.Type.PIPE) {
+            ProcessOutputLogger.logOutput(LOG, postmaster);
```

Comment:
? Guessing this does something special for piped output

Reply:
Guessing not necessary!  Read how the `ProcessOutputLogger` works.  : ]  The problem we were running into was that we were getting surefire failures due to a bug in their parsing of stuff logged to standard output.  So by sending this through the logger, we prevented Postgres and the control commands from interfering with the fork'd IPC.